### PR TITLE
N33: Release

### PR DESCRIPTION
**Pull request by AngelOnFira**
[Rendered version](https://agitated-saha-af60ca.netlify.app/news/033/): 1ca739c

_Part of #1010_

Todo:
- [x] Improve the name of IRL Events
- [x] Popular wg issues
- [x] Other news
- [x] Discussions
- [x] Contributions
- [x] Jobs
- [x] Bonus
- [x] Spelling + Grammar
- [ ] Link checks
- [x] Formatting

### CODE REVIEW

File: content/news/033/index.md [hunk 1]
```diff
@@ -31,8 +31,7 @@ Feel free to send PRs about your own projects!
 [Rust]: https://rust-lang.org
 [join]: https://github.com/rust-gamedev/wg#join-the-fun
 
-- [Rust GameDev Meetup](#rust-gamedev-meetup)
-- [Rust Graphics Meetup](#rust-graphics-meetup)
+- [IRL Events](#irl-events)
```
Comment (AngelOnFira) [inline content/news/033/index.md:34]: Thoughts on what this could be titled better as?

I'm thinking "Live events", or maybe "Bulletin board"

Comment (ozkriff) [reply content/news/033/index.md:34]: Originally, I intended to use custom top-level sections for "official announcements" from the Gamedev WG (events, our [blog posts](https://gamedev.rs/blog), podcast episodes, talk recordings, etc), but in practice "non-official" important stuff is posted like this as well. Maybe, just "Announcements" isn't that bad?

Comment (17cupsofcoffee) [reply content/news/033/index.md:34]: I like 'Announcements' 👍

Comment (AngelOnFira) [reply content/news/033/index.md:34]: Nice, I like it :)

**Comment by AngelOnFira** [comment]
Alright, we should be ready for final review now :)

File: content/news/033/index.md [hunk 2]
```diff
@@ -530,38 +532,35 @@ your project in an ergonomic manner without enforcing any structure or pattern,
...
-<!-- Up to 10 links to interesting issues -->
-
 ## Other News
 
 <!-- One-liners for plan items that haven't got their own sections. -->
 
 - Other game updates:
-  - [Last of the Sky Folk](https://ianjk.com/ld50)
-    is a LD50 a grapple-hook based platformer.
+  - [Last of the Sky Folk](https://ianjk.com/ld50) is a LD50 a grapple-hook
```
Comment (kettle11) [inline content/news/033/index.md:548]: Thanks for including this! 

This sentence seems to be missing a word. Here's a proposed fix:
 "[Last of the Sky Folk](https://ianjk.com/ld50) is a grapple-hook based platformer that's an entry for LD50."

Comment (AngelOnFira) [reply content/news/033/index.md]: Fixed :) also, if you're interested in showing if off at tomorrow's gamedev meetup, feel free to fill out the form!

https://forms.gle/BS1zCyZaiUFSUHxe6

**Review by 17cupsofcoffee — APPROVED** [review]
:+1: LGTM (don't forget to update the date before publishing!)

And sorry for not being around much this month, should be able to be more active for the May newsletter!

**Review by ozkriff — APPROVED** [review]

**Pull request merged**